recipe/meta.yaml

```diff
@@ -42,17 +40,17 @@ outputs:
       run:
         - python
         - deprecated >=1.2.13
+        - numba >=0.53
         - numpy >=1.21.0,<1.25
-        - pandas >=1.1.0,<1.6.0
-        - numba >=0.55
+        - pandas >=1.1.0,<2.0.0
         - scikit-learn >=0.24.0,<1.3.0
-        - statsmodels >=0.12.1
         - scipy >=1.2.0,<2.0.0
       run_constrained:
         - esig ==0.9.7
```

Comment:
Do we need a python selector here? [py<310]
https://github.com/sktime/sktime/blob/v0.17.0/pyproject.toml#L61

Reply:
I guess, **no**. Because those dependencies really have Python 3.11 support, for example, numba 0.57.0 has been released on [May 2, 2023](https://numba.readthedocs.io/en/stable/release-notes.html#version-0-57-0-1-may-2023), but sktime 0.17.0 was released on [April 4, 2023](https://github.com/sktime/sktime/commit/d9a038f9f259637aee5180bf7b0b420c93300e91). That's why they constrain python <3.11